views/Providers/Create/Form.tsx

```diff
@@ -1,27 +1,95 @@
 import { FC } from 'react'
 import { useForm } from 'react-hook-form'
 
-import { MutableProviderFormProps, ProviderEditable } from '@/services/GraphQL/providers/types'
+import Button from '@/components/Button'
+import btnStyles from '@/components/Button/Button.module.scss'
+import CustomInput from '@/components/CustomInput'
+import { MutableProviderUserFormProps, ProviderUserEditable } from '@/services/GraphQL/users/types'
 
-const CreateProviderForm: FC<MutableProviderFormProps> = ({ onSubmit, loading }) => {
+import styles from './ProviderCreateForm.module.scss'
+
+const CreateProviderForm: FC<MutableProviderUserFormProps> = ({ onSubmit, loading }) => {
   const {
     register,
     handleSubmit,
     formState: { errors },
-  } = useForm<ProviderEditable>()
+  } = useForm<ProviderUserEditable>()
   const submitHandler = handleSubmit(onSubmit)
-  const buttonText = loading ? 'Enviando' : 'Enviar'
+  const buttonText = loading ? 'Añadiendo' : 'Añadir'
 
   return (
-    <form onSubmit={submitHandler}>
-      <input type="text" placeholder="Comercial Name" {...register('commercialName', { required: true })} />
-      {errors.commercialName && <span>This field is required</span>}
-      <input type="text" placeholder="WhatsApp Phone" {...register('salesPhone')} />
-      {errors.salesPhone && <span>This field is required</span>}
-      <input type="number" min={0} placeholder="Order Index" {...register('orderIndex')} />
-      <input type="text" placeholder="Plan" {...register('plan')} />
-      <input type="text" placeholder="Logo URL" {...register('logoUrl')} />
-      <button type="submit">{buttonText}</button>
+    <form onSubmit={submitHandler} className={styles.container} noValidate>
+      <div className={styles['input-container']}>
+        <CustomInput
+          id="name"
+          placeholder="Escriba el nombre de la empresa"
+          label="Nombre de la empresa"
+          register={register}
+          name="commercialName"
+          required
+        >
+          {errors.commercialName && <span>El nombre es requerido.</span>}
+        </CustomInput>
+      </div>
+      <div className={styles['input-container']}>
+        <CustomInput
+          id="date"
+          placeholder="DD/MM/AA"
+          label="Fecha de inicio"
+          register={register}
+          name="createdAt"
+          required
+          type="date"
+        >
+          {errors.createdAt && <span>La fecha de inicio es requerido.</span>}
```

Comment:
```suggestion
          {errors.createdAt && <span>La fecha de inicio es requerida.</span>}
```

Reply:
Done.

Reply:
I don't see the change, can you check if you've pushed it, please?

Reply:
And now?